### ISSUE TITLE
government registers / ärende

### ISSUE THREAD

**Issue by salgo60** [govdirectory/website#159]
### The feature

1) from govdirectory find the government registers
2) search in all   goverment registers about a specific topic i.e. govdirectory is a hub that translate queries to the relevant language of the giverment and sends in a query
3) support also "non official" government registers

### Why?

**Background**: In Sweden www.datastory.org has built an application that makes it possible to search all government registers in Sweden

![image](https://user-images.githubusercontent.com/14206509/143723716-5ed3c6bd-2e01-434a-9e2e-d09df7ebb72b.png)

### Examples and inspiration

govdirectory should support links to websites as below
* Datastory:[open-registers-sweden](https://www.datastory.org/en/services/open-registers-sweden)
* Datastory:[sök i regeringens diarium](https://www.datastory.org/sv/services/open-registers-sweden)

**Comment by Abbe98** [comment]
Thank you for the link!

>  all government registers in Sweden

It's not clear to me what this service actually contains, it's definitely not all government registers in Sweden. 

Edit:
The "source" indicates that it's just the departments registers, which makes sense as these are open and easily accessible. Maybe we could find a way to link to each department.

**Comment by Ainali** [comment]
This could fit after we solve #102. 

But since this is not a global data source (right now only Sweden is available and the post don't mention expanding that), I don't think it has high priority. 

That said, pull requests are always welcome!

**Comment by Abbe98** [comment]
Yeah, a property for register/diarium might be a very good addition, although I think there is an official registry search we could point to.
